backend/src/tasks/pesyncdb.ts

```diff
@@ -14,6 +14,7 @@ const PE_DATA_SCHEMA = `
 -- Draft Database Schema to store scan data
 -- Includes Domain Masquerading, Credentals Exposed, Inffered Vulns, and Dark Web data
 
+
```

Comment:
What was this change for? Needed for dnstwist?

Reply:
Ya, these changes are needed to run dnstwist in the development environment:
- P&Es latest database schema
- Added sample organizations and domains